views/layout.pug

```diff
@@ -5,8 +5,13 @@ doctype html
 html
   head
     title= title
+    link(rel='stylesheet', href='https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css')
     link(rel='stylesheet', href='/stylesheets/style.css')
+
   body
+    script(src="https://code.jquery.com/jquery-3.3.1.slim.min.js", integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo", crossorigin="anonymous")
```

Comment:
According to [the bootstrap docs](https://getbootstrap.com/docs/4.1/getting-started/introduction/) these includes should go at the bottom of the body tag?

Reply:
I assume it's because you want to load the javascript last for faster page loading?

Reply:
Yes that's true. Sorry I totally missed that. Will be pushing new code within the next few minutes.